components/avatars/avatars.css

```diff
@@ -0,0 +1,26 @@
+
+.avatar{
+    width: 4rem;
+    height: 4rem;
+    /* border: solid; */
+    /* border-radius: 50%; */
+}
+.avatar>img{
+    border-radius: 50%;
+}
+.avatar.sm{
+    width: 3rem;
+    height: 3rem;
+}
+.avatar.med{
+    width: 5rem;
+    height: 5rem;
+}
+
+.avatar.larg{
+    width: 9rem;
+    height: 9rem;
+}
```

Comment:
`.larg` looks incomplete. You can name them in 2 letters sm, md, and lg or something that is more specific.

Reply:
I used this because after using many variations I finalized this one. So if it does looks that bad I would remove it and refactor the names

Reply:
Have changed the name and also done with naming .Thanks for reviewing